opencensus/stats/aggregation.py

```diff
@@ -51,8 +35,7 @@ class BaseAggregation(object):
     :param aggregation_type: represents the type of this aggregation
 
     """
-    def __init__(self, buckets=None, aggregation_type=Type.NONE):
-        self._aggregation_type = aggregation_type
+    def __init__(self, buckets=None):
         self._buckets = buckets or []
 
     @property
```

Comment:
You still have an aggregation_type property.

Reply:
I ended up removing `BaseAggregation` altogether, per @c24t's comments.